### PR TITLE
[dubbo] component spec

### LINKED ISSUE #2479: Add document for dubbo output binding
**What content needs to be created or modified?**
A component spec for the dubbo output binding component

**Describe the solution you'd like**
A component spec with a table describing metadata; possibly a usage example

**Where should the new material be placed?**
https://docs.dapr.io/reference/components-reference/supported-bindings/

**The associated pull request from dapr/dapr, dapr/components-contrib, or other Dapr code repos**
https://github.com/dapr/components-contrib/tree/master/bindings/dubbo
https://github.com/dapr/components-contrib/pull/1768

**Additional context**
Also add to the partial doc so it shows up on Bindings component spec landing page, here: https://github.com/dapr/docs/blob/v1.11/daprdocs/data/components/bindings/generic.yaml

### PR DESCRIPTION
**Pull request by hhunter-ms**
## Description

Taking a stab at the `bindings.alicloud.dubbo` component spec. 

## Issue reference

PR will close: #2479

### CODE REVIEW

**Comment by msfussell** [comment]
It is this metadata https://github.com/dapr/components-contrib/blob/99ce411ef80ea0afc4cb40228713ae4e2b152182/bindings/dubbo/context.go#L27

File: daprdocs/content/en/reference/components-reference/supported-bindings/dubbo.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+See [this](https://dubbo.apache.org/docs/) for the documentation for Alibaba Cloud Dubbo.
+
+```yml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: alicloud-dubbo-binding
+  namespace: default
+spec:
+  type: bindings.alicloud.dubbo
```
Comment (ItalyPaleAle) [inline daprdocs/content/en/reference/components-reference/supported-bindings/dubbo.md]: Should be just "bindings.dubbo". It's not an Alibaba-specific thing

File: daprdocs/content/en/reference/components-reference/supported-bindings/dubbo.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field              | Required | Binding support |  Details | Example |
+|--------------------|:--------:|------------|-----|---------|
+| `group` | N | Output |  | `""` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/dubbo.md:48]: Need examples and defaults here.

Comment (hhunter-ms) [reply daprdocs/content/en/reference/components-reference/supported-bindings/dubbo.md:48]: @LaurenceLiZhixin

**Comment by ItalyPaleAle** [comment]
Should be re-opened?

**Comment by msfussell** [comment]
@ItalyPaleAle @berndverst  - Can you help review and resolve this PR?

**Pull request closed without merging**